src/organisms/cards/SimpleFeaturedPolicyCard/index.js

```diff
@@ -33,53 +33,56 @@ function SimpleFeaturedPolicyCard(props) {
   return (
     <div className={classnames(...classes)}>
       <div className={styles['content']}>
-        { premium.price ?
+        { premium.price ? (
           <div className={styles['premium']}>
             <Text
               type={3}
               font='a'
             >
-              {formattedPremium} <Text tag='span' type={11} font='a' spaced color='neutral-2'>{`/${premium.format.toUpperCase()}`}</Text>
+              {formattedPremium}
+              {' '}
```

Comment:
how come there's an empty string element now?

Reply:
idk it's gone now